<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table_Main_Window</class>
 <widget class="QMainWindow" name="Table_Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Table_Main_Window</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="DeleteButton">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="AddButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>106</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Table"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDiagram"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_me"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Table">
   <property name="text">
    <string>New Table</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="actionDiagram">
   <property name="text">
    <string>Diagram</string>
   </property>
  </action>
  <action name="actionInsert_Row">
   <property name="text">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="actionDelete_Row">
   <property name="text">
    <string>Delete Row</string>
   </property>
  </action>
  <action name="actionAbout_me">
   <property name="text">
    <string>about me</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>AddButton</sender>
   <signal>pressed()</signal>
   <receiver>Table_Main_Window</receiver>
   <slot>addButtonSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteButton</sender>
   <signal>pressed()</signal>
   <receiver>Table_Main_Window</receiver>
   <slot>deleteButtonSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addButtonSlot()</slot>
  <slot>deleteButtonSlot()</slot>
 </slots>
</ui>
